utils/sample-createspot.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+##  This is sample code that will generally show you how to launch a spot instance on aws and leverage the 
+##  automation built into deepracer-for-cloud to automatically start training
+##  Changes required to work:
+##     Input bucket info
+##     Input security group, iam role, and key-name
+
+## BUCKET is the fundamental input. you need to tell it where in s3 your training will be stored
+## just the bucket name.  no S3:// etc
+
+BUCKET=<########>
```

Comment:
Would be good with Bucket and Prefix; to let people store it somewhere else than the root.

Reply:
this has been updated to include bucket and/or bucket and prefix.